content/hi/docs/concepts/architecture/leases.md

```diff
@@ -0,0 +1,102 @@
+---
+title: पट्टों
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+वितरित प्रणालियों को अक्सर पट्टों की आवश्यकता होती है, जो साझा संसाधनों को लॉक करने और एक सेट के सदस्यों के बीच गतिविधि को समन्वयित करने के लिए एक तंत्र प्रदान करते हैं।
+कुबेरनेट्स में, पट्टे की अवधारणा को [लीज](/docs/reference/kubernetes-api/cluster-resources/lease-v1/) द्वारा दर्शाया गया है।
+`coordination.k8s.io` में ऑब्जेक्ट्स {{< glossary_tooltip text="API समूह" term_id="api-group" >}},
+जिनका उपयोग सिस्टम-महत्वपूर्ण क्षमताओं जैसे नोड हर्टबीट और घटक-स्तरीय नेता चुनाव के लिए किया जाता है।
+
+<!-- body -->
+
+## नोड दिल की धड़कन {#नोड दिल की धड़कन}
```

Comment:
On the fence about calling it  दिल की धड़कन because the following sentences are awfully hard for any normal Hindi reader to read, let alone comprehend.

Reply:
Can we start a discussion about this on the #kubernetes-docs-hi slack channel?